pom.xml

```diff
@@ -115,6 +115,7 @@
     <module>server</module>
     <module>client</module>
     <module>integration-test/common</module>
+    <module>metrics-prometheus</module>
```

Comment:
Why do we add this module twice?

Reply:
We have chosen prometheus as our project default metric system. In our origin implement, our code use prometheus everywhere. Could we remove this module? Could we put the promethesus to the `common` module, too?